pipeline_steps/deploy.groovy

```diff
@@ -55,11 +55,22 @@ def upgrade(String stage_name, String upgrade_script, List env_vars) {
           sh "git reset --hard"
         }
         common.prepareRpcGit()
```

Comment:
the multi-node-aio repo been copied into a virtual machine names deploy1, not the rpc-openstack, the baremetal will be just like a switch/router, and all those jobs are done within the virtual machines.